src/js/modules/RoutingApp.js

```diff
@@ -48,12 +51,14 @@ const RoutingApp = () => {
   };
   return (
     <Provider store={store}>
-      <ConnectedRouter history={appHistory}>
-        <Switch>
-          <Route exact path="/" component={HomePage}/>
-          { createSectionRoutes() }
-          { createArticleRoutes() }
-        </Switch>
+       <ConnectedRouter history={appHistory}>
+        <Layout>

```

Comment:
your code is clearly untested: <Layout> is undefined here. please test before committing.